### PR TITLE
Add socratic 97

### PR DESCRIPTION
**Pull request by btsea**
Do not merge until content is complete

### CODE REVIEW

**Comment by jnewbery** [comment]
pushed a small fixup commit. Will put together my links next

File: _posts/2019-10-24-socratic-seminar-97.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+- [Newsletter #64](https://bitcoinops.org/en/newsletters/2019/09/18/)
+- [Newsletter #65](https://bitcoinops.org/en/newsletters/2019/09/25/)
+- [Newsletter #66](https://bitcoinops.org/en/newsletters/2019/10/02/)
+- [Newsletter #67](https://bitcoinops.org/en/newsletters/2019/10/09/)
+- [Newsletter #68](https://bitcoinops.org/en/newsletters/2019/10/16/)
+
+## Network Data
+
+- [Six Addresses Are Dominating Ethereum (ETH) Usage Right Now](https://bitcoinist.com/six-addresses-are-dominating-ethereum-eth-usage-right-now/)
```
Comment (jnewbery) [inline _posts/2019-10-24-socratic-seminar-97.md]: I don't trust bitcoinist as a news source

File: _posts/2019-10-24-socratic-seminar-97.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+- [Short Paper: Towards Characterizing Sybil Attacks in Cryptocurrency Mixers](https://eprint.iacr.org/2019/1111)
+- [Simplicity's first transaction](https://lists.ozlabs.org/pipermail/simplicity/2019/000018.html)
+- [Transparent SNARKs from DARK Compilers](https://eprint.iacr.org/2019/1229.pdf)
+- [Binary quadtratic forms](http://diyhpl.us/%7Ebryan/papers2/bitcoin/vdf-competition-classgroups-binary-quadratic-form.pdf)
+- [Bitcoin Chain Width ExpansionDenial-of-Service Attacks](https://bcoin.io/papers/bitcoin-chain-expansion.pdf)
+
+
+#### InfoSec
+
+- [lnd, c-lightning, eclair CVE-2019-12998 / CVE-2019-12999 /	CVE-2019-13000](https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-September/002174.html) [Rusty broadcast](https://twitter.com/i/broadcasts/1yqkdnryandjv)
```
Comment (jnewbery) [inline _posts/2019-10-24-socratic-seminar-97.md]: It's difficult to see that this is two links. I suggest making the second link a sub-bullet.

Comment (jnewbery) [reply _posts/2019-10-24-socratic-seminar-97.md]: Also, rusty broadcast is broken.

File: _posts/2019-10-24-socratic-seminar-97.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+#### lnd
+
+- [0.8.0-beta](https://github.com/lightningnetwork/lnd/releases/tag/v0.8.0-beta)
+
+#### c-lightning
+
+#### BOLTs
+
+- Rusty Russell BOLT Broadcasts:
+  - [BOLT 4](https://twitter.com/i/broadcasts/1drkzlvmoldjb)
```
Comment (jnewbery) [inline _posts/2019-10-24-socratic-seminar-97.md]: all of these links are broken except **BOLT 7 part 1**

**Comment by jnewbery** [comment]
I've pushed the PR links.

File: _posts/2019-10-24-socratic-seminar-97.md (hunk 1 above)
Comment (btsea) [reply _posts/2019-10-24-socratic-seminar-97.md]: Alright let's remove it.

File: _posts/2019-10-24-socratic-seminar-97.md (hunk 2 above)
Comment (btsea) [reply _posts/2019-10-24-socratic-seminar-97.md]: Changed to direct links to the tweets, the broadcast links appear unreliable.

File: _posts/2019-10-24-socratic-seminar-97.md (hunk 3 above)
Comment (btsea) [reply _posts/2019-10-24-socratic-seminar-97.md]: Fixing, thanks.

**Pull request merged**